verilog/analysis/checkers/parameter_name_style_rule_test.cc

```diff
@@ -63,16 +60,10 @@ TEST(ParameterNameStyleRuleTest, RejectTests) {
   const std::initializer_list<LintTestCase> kTestCases = {
       {"module foo; localparam ", {kToken, "Bar_Hello"}, " = 1; endmodule"},
       {"module foo; localparam int ", {kToken, "Bar_Hello"}, " = 1; endmodule"},
-      {"module foo; localparam type ",
-       {kToken, "Bar_Hello_1"},
-       " = 1; endmodule"},
       {"module foo; parameter int ", {kToken, "__Bar"}, " = 1; endmodule"},
       {"module foo #(parameter int ",
        {kToken, "Bar_1_Hello"},
        " = 1); endmodule"},
-      {"module foo #(parameter type ",
-       {kToken, "Bar_1_Hello__"},
-       "); endmodule"},
```

Comment:
Move these to `AcceptTests` instead of deleting them?

Reply:
Sure!